lib/Convos.pm

```diff
@@ -69,6 +69,7 @@ sub startup {
   # Add basic routes
   $r->get('/')->to(template => 'index')->name('index');
   $r->get('/err/500')->to(cb => sub { die 'Test 500 page' });
+  $r->get('/sw' =>  [ format => 'js']);
```

Comment:
Could we name it "service-worker" instead of "sw"? "sw" for me could be star-wars.js 😆